assets/javascripts/modules/analytics/ga.js

```diff
@@ -16,6 +18,10 @@ define(['utils/cookie',
         })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
         /*eslint-enable */
 
+        // Do JellyFish first as they don't want our custom dimensions
+        ga('create', 'UA-44575989-1', 'auto', 'jellyfishGA');
```

Comment:
We have to make 2 separate calls they don't combine. JellyFish's tracker also redirects 3 times...

https://developers.google.com/analytics/devguides/collection/analyticsjs/creating-trackers#working_with_multiple_trackers